plugin/trino-hudi/src/main/java/io/trino/plugin/hudi/util/HudiAvroSerializer.java

```diff
@@ -96,25 +113,29 @@ public class HudiAvroSerializer
     private final List<HiveColumnHandle> columnHandles;
     private final List<Type> columnTypes;
     private final Schema schema;
+    private final ZoneId zoneId;
 
     public HudiAvroSerializer(List<HiveColumnHandle> columnHandles)
     {
         this.columnHandles = columnHandles;
         this.columnTypes = columnHandles.stream().map(HiveColumnHandle::getType).toList();
         // Fetches projected schema
         this.schema = constructSchema(columnHandles.stream().filter(ch -> !ch.isHidden()).map(HiveColumnHandle::getName).toList(),
-                columnHandles.stream().filter(ch -> !ch.isHidden()).map(HiveColumnHandle::getHiveType).toList(), false);
+                columnHandles.stream().filter(ch -> !ch.isHidden()).map(HiveColumnHandle::getHiveType).toList());
         this.synthesizedColumnHandler = null;
+        this.zoneId = ZoneId.systemDefault();
     }
 
-    public HudiAvroSerializer(List<HiveColumnHandle> columnHandles, SynthesizedColumnHandler synthesizedColumnHandler)
+    public HudiAvroSerializer(List<HiveColumnHandle> columnHandles, SynthesizedColumnHandler synthesizedColumnHandler, ConnectorSession session)
```

Comment:
Can remove this as it's used to get `zoneId` which I thought was required to convert false UTC to true UTC